sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -517,6 +517,16 @@ public Node getNode() {
         }
 
         public ClientCall<ProtoRequestT, ResponseT> createCall() {
+            // SECTION 1
+            System.out.format("Node: %s", node.address != null ? node.address.getAddress().toString() : "NULL");
```

Comment:
Since this seems to be copy and pasted a couple times maybe we want to create a logging method instead and just invoke that?

Reply:
I agree that would be much better.